lib/activerecord-import/import.rb

```diff
@@ -350,12 +350,7 @@ def import_helper( *args )
           # this next line breaks sqlite.so with a segmentation fault
           # if model.new_record? || options[:on_duplicate_key_update]
           column_names.map do |name|
-            name = name.to_s
-            if respond_to?(:defined_enums) && defined_enums.key?(name) # ActiveRecord 5
```

Comment:
This logic doesn't seem to be necessary if models aren't converted to arrays and then back to models.